airbyte-config/models/src/main/resources/types/ActorDefinitionResourceRequirements.yaml

```diff
@@ -0,0 +1,27 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/ActorDefinitionResourceRequirements.yaml
+title: ActorDefinitionResourceRequirements
+description: actor definition specific resource requirements
+type: object
+additionalProperties: true
+properties:
+  default:
+    description: if set, these are the requirements that should be set for ALL jobs run for this actor definition.
+    "$ref": ResourceRequirements.yaml
+  jobSpecific:
```

Comment:
want to leave this open to handling requirement for other job types (e.g. check, or what now), but for right now the acute need is for sync jobs.